Added my Balance 12 layout for Atreus

yttyx opened the pull request:
The first Balance 12 layout in the QMK repository?

drashna commented on keyboards/atreus/keymaps/yttyx/config.h:
```diff
@@ -0,0 +1,59 @@
+#ifndef CONFIG_H
```
You shouldn't need all of this.  Only the stuff that you've customized.

drashna commented on keyboards/atreus/keymaps/yttyx/keymap.c:
```diff
@@ -0,0 +1,126 @@
+  )
+};
+
+const uint16_t PROGMEM fn_actions[] = {};
```
If you've disabled these features, you should remove these functions

drashna commented on keyboards/atreus/keymaps/yttyx/README.md:
```diff
@@ -0,0 +1,81 @@
+
+## To flash
+
+> avrdude -p atmega32u4 -c avr109 -U flash:w:atreus_yttyx.hex -P /dev/ttyACM0
```
`make atreus:yttyx:avrdude` should take care of this automatically (include port detection, IIRC)

yttyx commented:
Thank you! - I have made the following changes:
- Removed unnecessary config.h lines
- Removed unused stub functions from keymap.c
- Modified layout build instructions

drashna commented:
Awesome, thanks!  Looks good!

yttyx commented:
Much appreciated - thank you!
